src/shared/fs/templateRegistry.ts

```diff
@@ -120,11 +121,11 @@ export class AsyncCloudFormationTemplateRegistry {
                         // Allows for new message to be created if templateRegistry variable attempted to be used again
                         this.setupProgressMessage = undefined
                     })
```

Comment:
@nkomonen-amazon Possible bug: seems like Cancel does not actually cancel the work being done by https://github.com/aws/aws-toolkit-vscode/blob/a0a9280060e17ac8983cc4a6be83c28656011955/src/shared/cloudformation/activation.ts#L68-L69